src/ducks/notifications/helpers.spec.js

```diff
@@ -0,0 +1,13 @@
+import { getBanksUrl } from './helpers'
+
+describe.only('getBanksUrl', () => {
+  it('should should append "-banks" to the first component', () => {
+    expect(getBanksUrl('test.cozy.works')).toBe('test-banks.cozy.works/#/')
```

Comment:
The URL would normally contain `https://` right ? If so, I think it should be also in the test